go-controller/pkg/ovn/controller/services/node_tracker.go

```diff
@@ -262,7 +266,7 @@ func (nt *nodeTracker) updateNode(node *v1.Node) {
 		grName,
 		chassisID,
 		l3gatewayAddresses,
-		hostAddressesIPs,
+		ovnManagedAddresses,
```

Comment:
Why is nodeTracker only handling ovn managed addresses now?

Reply:
Mistake - reverted